samples/graph-activity-feed/nodejs/appManifest/manifest.json

```diff
@@ -1,110 +1,139 @@
 {
-  "$schema": "https://developer.microsoft.com/json-schemas/teams/v1.19/MicrosoftTeams.schema.json",
-  "manifestVersion": "1.19",
+  "$schema": "https://developer.microsoft.com/en-us/json-schemas/teams/vdevpreview/MicrosoftTeams.schema.json",
+  "manifestVersion": "devPreview",
   "version": "1.0.0",
   "id": "${{TEAMS_APP_ID}}",
   "developer": {
-    "name": "Microsoft",
-    "websiteUrl": "https://www.teams.com",
-    "privacyUrl": "https://www.teams.com/privacy",
-    "termsOfUseUrl": "https://www.teams.com/termsofuse"
+      "name": "Microsoft",
+      "websiteUrl": "https://www.teams.com",
+      "privacyUrl": "https://www.teams.com/privacy",
+      "termsOfUseUrl": "https://www.teams.com/termsofuse"
   },
   "icons": {
-    "color": "color.png",
-    "outline": "outline.png"
+      "color": "color.png",
+      "outline": "outline.png"
   },
   "name": {
-    "short": "NotifyFeedApp",
-    "full": "NotifyFeedApp for Send Activity Feed"
+      "short": "NotifyFeedApp_1",
+      "full": "NotifyFeedApp for Send Activity Feed"
   },
   "description": {
-    "short": "Sample app that uses Graph API to send activity feed notifications in Teams.",
-    "full": "This Microsoft Teams sample app demonstrates how to send activity feed notifications using the Microsoft Graph API in a Teams Tab."
+      "short": "Sample app that uses Graph API to send activity feed notifications in Teams.",
+      "full": "This Microsoft Teams sample app demonstrates how to send activity feed notifications using the Microsoft Graph API in a Teams Tab."
   },
   "accentColor": "#235EA5",
   "configurableTabs": [
-    {
-      "configurationUrl": "https://${{TAB_DOMAIN}}/configure",
```

Comment:
Why did you update TAB_DOMAIN to BOT_DOMAIN? The sample doesn't work if you use TAB_DOMAIN.